srcpkgs/pinentry/template

```diff
@@ -1,13 +1,14 @@
 # Template file for 'pinentry'
 pkgname=pinentry
 version=1.1.0
-revision=1
+revision=2
 build_style=gnu-configure
 configure_args="--disable-rpath --without-libcap --disable-pinentry-gtk
  --enable-pinentry-curses --enable-fallback-curses --enable-pinentry-gtk2
- --enable-pinentry-emacs --enable-pinentry-qt --enable-pinentry-tty"
+ --enable-pinentry-emacs --enable-pinentry-qt --enable-pinentry-tty
+ --enable-pinentry-gnome3"
 hostmakedepends="pkg-config"
-makedepends="ncurses-devel gtk+-devel libassuan-devel libgpg-error-devel qt5-devel"
+makedepends="ncurses-devel gcr-devel gtk+-devel libassuan-devel libgpg-error-devel qt5-devel"
 short_desc="PIN or passphrase entry dialogs for GnuPG"
 maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="GPL-2"
```

Comment:
while you're at it please use the SPDX short-identifier

Reply:
Like that?

Reply:
Did I do it correctly?

Reply:
Yes like that but the license is GPL-2.0-or-later due to the presence of

```
either version 2 of the License, or * (at your option) any later version.  
```